app/views/foods/_food.html.erb

```diff
@@ -1,27 +1,38 @@
-<div id="<%= dom_id food %>">
-  <p>
-    <strong>Name:</strong>
-    <%= food.name %>
-  </p>
+<%# <table id="foods-table">
+  <tr>
+    <th>
+    Food
+    </th>
 
-  <p>
-    <strong>Measurementunit:</strong>
-    <%= food.measurementUnit %>
-  </p>
+    <th>
+    Measurement unit
+    </th>
+    <th>
+    Unit price
+    </th>
+     <th>
+    Actions
+    </th>
+    
+   
+  </tr>
+
+  <tr>
+    <td>
+     <%= food.name %>
+    </td>
 
-  <p>
-    <strong>Price:</strong>
+    <td>
+    <%= food.measurementUnit %>
+    </td>
+    <td>
     <%= food.price %>
-  </p>
+    </td>
+     <td>
+    Delete
+    </td>
+  <%# </tr> %>
 
```

Comment:
- [ ] When I tried to delete any of the food I created, I got the error message below so kindly fix this as well so that when a user adds foods, they are able to delete it when it's no longer needed :+1: 
- Error Message
![image](https://github.com/ridabensalem/recipe-app/assets/102699958/9ad30b08-ecc0-466c-ae74-a02da5e6d558)

Reply:
Fixed.